src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -700,6 +705,12 @@ namespace PurchasedAirManager {
 
 			SetupOutputVariable( "Zone Ideal Loads Hybrid Ventilation Available Status []", PurchAir( PurchAirNum ).AvailStatus, "System", "Average", PurchAir( PurchAirNum ).Name );
 
+			//air flows
```

Comment:
And it doesn't render that way for me when I look in Chrome instead of IE.  Fascinating.  Never mind.